docs/quick_start.md

```diff
@@ -0,0 +1,153 @@
+
+▶ docker ps
+CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                NAMES
+5d6938439d50        centos/httpd        "/run-httpd.sh"          3 seconds ago       Up 1 seconds        0.0.0.0:80->80/tcp   default_centos-httpd_ec75a2fe2a50
+```
+
+__Runnning Apache on Kubernetes:__
+```sh
+▶ sudo atomicapp run projectatomic/helloapache --provider=kubernetes
+2016-02-25 15:03:04,341 - [INFO] - main.py - Action/Mode Selected is: run
+2016-02-25 15:03:04,343 - [INFO] - base.py - Unpacking image: projectatomic/helloapache to /var/lib/atomicapp/projectatomic-helloapache-c0dd79b5e757
+2016-02-25 15:03:07,983 - [INFO] - container.py - Skipping pulling Docker image: projectatomic/helloapache
+2016-02-25 15:03:07,984 - [INFO] - container.py - Extracting nulecule data from image: projectatomic/helloapache to /var/lib/atomicapp/projectatomic-helloapache-c0dd79b5e757
+886e10a3244f982f3302ab9058ab7b377c6f83e2cf63f001e1ba011358d0b471
+2016-02-25 15:03:08,332 - [INFO] - kubernetes.py - Using namespace default
+2016-02-25 15:03:08,332 - [INFO] - kubernetes.py - trying kubectl at /usr/bin/kubectl
+2016-02-25 15:03:08,332 - [INFO] - kubernetes.py - trying kubectl at /usr/local/bin/kubectl
+2016-02-25 15:03:08,332 - [INFO] - kubernetes.py - found kubectl at /usr/local/bin/kubectl
+2016-02-25 15:03:08,332 - [INFO] - kubernetes.py - Deploying to Kubernetes
+...
+
+Your application resides in /var/lib/atomicapp/projectatomic-helloapache-c0dd79b5e757
+Please use this directory for managing your application
+
+▶ kubectl get po
+NAME                   READY     STATUS    RESTARTS   AGE
+helloapache            1/1       Running   0          2m
+k8s-etcd-127.0.0.1     1/1       Running   0          1d
+k8s-master-127.0.0.1   4/4       Running   0          1d
+k8s-proxy-127.0.0.1    1/1       Running   0          1d
+```
+
+__Fetch, edit and run Apache on Kubernetes:__
+```sh
+▶ mkdir ./localdir
+
+▶ sudo atomicapp fetch projectatomic/helloapache --destination ./localdir/
+2016-02-25 15:35:41,439 - [INFO] - main.py - Action/Mode Selected is: fetch
+2016-02-25 15:35:41,440 - [INFO] - base.py - Unpacking image: projectatomic/helloapache to helloapache
+2016-02-25 15:35:45,067 - [INFO] - container.py - Skipping pulling Docker image: projectatomic/helloapache
+2016-02-25 15:35:45,067 - [INFO] - container.py - Extracting nulecule data from image: projectatomic/helloapache to helloapache
+c12d2047fab44f2906b9cbee3ac86c6c6499921ce33a90085e8765491b44f447
+
+Your application resides in localdir
+Please use this directory for managing your application
+
+▶ cd localdir
+
+▶ cat Nulecule
+...
+      - name: hostport
+        description: The host TCP port as the external endpoint
+        default: 80
+...
+
+▶ vim Nulecule # edit port 80 to 8080
+
+▶ cat Nulecule 
+...
+      - name: hostport
+        description: The host TCP port as the external endpoint
+        default: 8080
```

Comment:
I agree with that. This is just one way that a user can package a container up.

I'll add something in regards to using an answers.conf as well.